linode/manager
# upcoming: [M3-7733] - Access key "Permissions" drawer ARIA label needs to be updated for OBJ Multicluster

## Description 📝
Render BucketPermissionsTable when OBJ Multicluster flag is enabled.

## Preview 📷
![image](https://github.com/linode/manager/assets/119517080/d81d60ad-452b-428f-885e-ddc3a5a0e5cc)


## How to test 🧪

### Prerequisites
(How to setup test environment)
- Turn on OBJ Multicluster flag
- Turn on MSW

### Verification steps
(How to verify changes)
- Navigate to http://localhost:3000/object-storage/access-keys
- Click on "Permissions" from the menu.
- Verify ARIA labels for checkboxes (Should use region instead of cluster). 

## As an Author I have considered 🤔

*Check all that apply*


---
